go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/livekit/server-sdk-go/v2
 
-go 1.23
+go 1.22.7
```

Comment:
was this intentional?

Reply:
yes, it was raised unintentionally in protocol and was never rolled back in the SDK side